code/modules/power/battery.dm

```diff
@@ -225,8 +225,8 @@ var/global/list/battery_online =	list(
 	var/data[0]
 	data["nameTag"] = name_tag
 	data["storedCapacity"] = round(100.0*charge/capacity, 0.1)
-	data["charge"] = charge
-	data["capacity"] = capacity
+	data["charge"] = charge/SMESRATE //Compensates for the input/output rate
+	data["capacity"] = capacity/SMESRATE
```

Comment:
I'm not sure this wants to be modified at all

Reply:
It works without any issue in-game but maybe it's not a good idea to divide something that could be 0.